core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -170,10 +170,9 @@
   // general properties
   GENERAL_PREFIX("general.", null, PropertyType.PREFIX,
       "Properties in this category affect the behavior of accumulo overall, but do not have to be consistent throughout a cloud."),
-  GENERAL_CLASSPATHS(AccumuloClassLoader.CLASSPATH_PROPERTY_NAME, AccumuloClassLoader.ACCUMULO_CLASSPATH_VALUE, PropertyType.STRING,
+  GENERAL_CLASSPATHS(AccumuloClassLoader.CLASSPATH_PROPERTY_NAME, "", PropertyType.STRING,
```

Comment:
I think it should be deprecated. I think users should avoid setting up the classpath in accumulo-site.xml.